src/test/pages/LecturePage.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount, shallow } from 'enzyme';
+import './loadSubject.mock';
+import { LecturePage } from '../../main/pages/LecturePage';
+
+describe('LecturePage', () => {
+    let renderedComponent;
+
+    const props = {
+        loadSubject,
+        match: { path: '/courses/:subject_id/:subject', url: '/courses/2D0MoB57yByiAQhLSGnK/MQMO', isExact: true, params: { subject_id: '2D0MoB57yByiAQhLSGnK', subject: 'MQMO' } },
+        location: { pathname: '/courses/2D0MoB57yByiAQhLSGnK/MQMO', search: '', hash: '' },
+        history: { length: 2, action: 'POP', location: { pathname: '/courses/2D0MoB57yByiAQhLSGnK/MQMO', search: '', hash: '' } },
```

Comment:
I don't think ```location``` and ```history``` are needed here (if you don't test ```BaseLayout``` component in this test suit) ^^' so you can remove those lines